lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -138,19 +186,13 @@ class OffscreenImages extends ByteEfficiencyAudit {
     }, /** @type {Map<string, WasteResult>} */ (new Map()));
 
     const settings = context.settings;
-    return artifacts.requestFirstCPUIdle({trace, devtoolsLog, settings}).then(firstInteractive => {
-      // The filter below is just to be extra safe that we exclude images that were loaded post-TTI.
-      // If we're in the Lantern case and `timestamp` isn't available, we just have to rely on the
-      // graph simulation doing the right thing.
-      const ttiTimestamp = firstInteractive.timestamp ? firstInteractive.timestamp / 1e6 : Infinity;
-
-      const items = Array.from(resultsMap.values()).filter(item => {
-        const isWasteful =
-          item.wastedBytes > IGNORE_THRESHOLD_IN_BYTES &&
-          item.wastedPercent > IGNORE_THRESHOLD_IN_PERCENT;
-        const loadedEarly = item.requestStartTime < ttiTimestamp;
-        return isWasteful && loadedEarly;
-      });
+    return artifacts.requestInteractive({trace, devtoolsLog, settings}).then(interactive => {
+      const unfilteredResults = Array.from(resultsMap.values());
+      const lanternInteractive = /** @type {LH.Artifacts.LanternMetric} */ (interactive);
+      // Filter out images that were loaded after all CPU activity
+      const items = interactive.timestamp ?
```

Comment:
shouldnt we base this on settings.throttlingMethod or something?

Reply:
done